yaptide/celery/tasks.py

```diff
@@ -115,57 +115,28 @@ def simulation_task_status(task_id: str) -> dict:
 
     result = {
         'status': 'OK',
-        'message': {
-            'state': task.state
+        'content': {
+            'state': task.state,
```

Comment:
```suggestion
            'state': task.state
```

do we need this comma ?

Reply:
@grzanka we do not, but this is convention used everywhere in the project as in converter repository.

Reply:
I am not commiting this suggestion

Reply:
ok, no problem